onnx/defs/parser.cc

```diff
@@ -53,7 +56,40 @@ Status ParserBase::Parse(Literal& result) {
       }
     } else
       result.value = std::string(from + 1, next_ - from - 2); // skip enclosing quotes
-  } else if ((isdigit(nextch) || (nextch == '-'))) {
+      return Status::OK();
```

Comment:
Unindent this line? The CI is failing because of this, which does look suspect (unclear whether the return was intended to be part of the else or not).